llama_index/vector_stores/pinecone.py

```diff
@@ -129,11 +132,7 @@ def __init__(
         index_name: Optional[str] = None,
         environment: Optional[str] = None,
         namespace: Optional[str] = None,
-        metadata_filters: Optional[Dict[str, Any]] = None,
```

Comment:
Much of the LlamaIndex docs still refer to the legacy approach for adding metadata_filters. The docs are now inaccurate and misleading. For example:

https://gpt-index.readthedocs.io/en/latest/how_to/integrations/vector_stores.html